instrumentation/okhttp/okhttp-3.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/okhttp/v3_0/OkHttp3Singletons.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.okhttp.v3_0;
+
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.instrumentation.okhttp.v3_0.OkHttpTracing;
+import io.opentelemetry.instrumentation.okhttp.v3_0.internal.OkHttpNetAttributesExtractor;
+import io.opentelemetry.javaagent.instrumentation.api.instrumenter.PeerServiceAttributesExtractor;
+import okhttp3.Interceptor;
+
+/** Holder of singleton interceptors for adding to instrumented clients. */
+public final class OkHttp3Singletons {
+
+  public static final Interceptor TRACING_INTERCEPTOR =
+      OkHttpTracing.newBuilder(GlobalOpenTelemetry.get())
+          .addAttributesExtractor(
+              PeerServiceAttributesExtractor.create(new OkHttpNetAttributesExtractor()))
```

Comment:
Is there a reason why we wouldn't want these to be included by default, in library instrumentation usages?

Reply:
Is that the mapping between ip and name? It might be nice to expose that option to the library instrumentation as well in the future.

Reply:
I think the idea was that library instrumentation users have more convenient ways to set `peer.service` than using a global map, since they can customize interceptors/extractors at different usage points

Reply:
true, but they would have to understand the otel convention around peer.service to even know that it's a thing to set.

Reply:
opened #3560 to track since this PR is just following existing convention